data/Templates/Hl7v2/CodeSystem/CodeSystem.json

```diff
@@ -1979,8 +1985,13 @@
                 "system": "http://hl7.org/fhir/contact-point-use"
             }, 
             "EMR": {
-                "code": "", 
-                "display": "", 
+                "code": "emergency", 
+                "display": "Emergency", 
+                "system": ""
+            }, 
+            "FAX": {
```

Comment:
Would you mind pasting your reference links/resources here?

Reply:
A [pull request](https://github.com/microsoft/FHIR-Converter/pull/253) is created to request the merge into the dotliquid branch. Note that I use the same contact-point-use FHIR reference although EMR is yet to be included. My company has a need to label a phone number as emergency contact. The interesting part is that EMR is included in Microsoft template but not added to this file. This is what I add it to enable this feature.